graph/Cargo.toml

```diff
@@ -33,9 +33,9 @@ slog = { version = "2.2.3", features = ["release_max_level_debug", "max_level_tr
 slog-async = "2.3.0"
 slog-envlogger = "2.1.0"
 slog-term = "2.4.0"
-tiny-keccak = "1.0"
+tiny-keccak = "1.4.2"
 tokio = "0.1.11"
 tokio-executor = "0.1.5"
 tokio-retry = "0.2"
 tokio-timer = "0.2.7"
-web3 = "0.5.0"
+web3 = { git = "https://github.com/olibearo/rust-web3", branch = "trace-struct" }
```

Comment:
Last week I tried to switch to upstream, it generated a lot of compilation errors since upstream made a lot of changes to the `ethereum-type`s. I punted since it was late on Friday, I'll give this another shot today to answer your question